encrypted-media/scripts/setmediakeys-multiple-times-with-different-mediakeys.js

```diff
@@ -53,34 +53,35 @@ function runTest(config, qualifier) {
             // Set mediaKeys2 on video element (switching MediaKeys) need not
```

Comment:
This could probably be clearer. For example, "It is up the the implementation whether to allow the MediaKeys to be changed after setting the source. The following step handles both cases."